api.json

```diff
@@ -730,28 +775,33 @@
               },
               "web_push_topic": {
                 "type": "string",
-                "description": "Channel: Push Notifications\nPlatform: All Browsers\nDisplay multiple notifications at once with different topics.\n"
+                "description": "Channel: Push Notifications\nPlatform: All Browsers\nDisplay multiple notifications at once with different topics.\n",
+                "nullable": true
               },
               "apns_alert": {
                 "type": "object",
                 "description": "Channel: Push Notifications\nPlatform: iOS 10+\niOS can localize push notification messages on the client using special parameters such as loc-key. When using the Create Notification endpoint, you must include these parameters inside of a field called apns_alert. Please see Apple's guide on localizing push notifications to learn more.\n",
-                "writeOnly": true
+                "writeOnly": true,
+                "nullable": true
               },
               "send_after": {
-                "type": "string",
-                "format": "date-time",
+                "type": "integer",
+                "format": "int64",
```

Comment:
should stay string

Reply:
in the future, instead of a commit reverting the previous history, try modifying the initial commit and removing the change.